src/assets/modules/data/README.md

```diff
@@ -0,0 +1,35 @@
+# Data module
+## Quick Summary
+Data is a module that lets you fetch data from CKAN (data.qld.gov.au) through PostgreSQL queries 
+
+## Dependencies
```

Comment:
I think we need to make everything production ready. This SWE3 project can get a green light any day. 
Please have a discussion with @raymedia to see the timelines.